backend/src/main/java/pl/edu/agh/io/backend/JsonConfig.java

```diff
@@ -0,0 +1,24 @@
+package pl.edu.agh.io.backend;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.NoArgsConstructor;
+import lombok.extern.jackson.Jacksonized;
+
+@NoArgsConstructor
+@Jacksonized
+public class JsonConfig {
+    private String endpointURL;
+
+    public JsonConfig(String endpointURL) {
+        this.endpointURL = endpointURL;
+    }
+
+    public String getEndpointURL() {
+        return endpointURL;
+    }
+
+    public void setEndpointURL(String endpointURL) {
+        this.endpointURL = endpointURL;
+    }
+}
```

Comment:
Do zastąpienia rekordem

Reply:
Właśnie tutaj możliwe, że się nie da, bo klasa jest deserializowana i jest do niej dodana anotacja NoArgsConstructor, przez co niestety rekord nie działa. Za to trochę skróciłem klasę, wywalając z niej getter i konstruktor, bo nie są potrzebne.